integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -463,17 +463,8 @@ def test_hash_groupby_single_distinct_collect(data_gen):
         df_fun=lambda spark: gen_df(spark, data_gen, length=100),
         table_name="tbl", sql=sql)
 
-# Queries with multiple distinct aggregations will fallback to CPU if they also contain
-# collect aggregations. Because Spark optimizer will insert expressions like `If` and `First`
-# when rewriting distinct aggregates, while `GpuFirst` doesn't support the datatype of collect
-# aggregations (ArrayType).
-# TODO: support GPUFirst on ArrayType https://github.com/NVIDIA/spark-rapids/issues/3097
 @approximate_float
 @ignore_order(local=True)
-@allow_non_gpu('SortAggregateExec',
-               'SortArray', 'Alias', 'Literal', 'First', 'If', 'EqualTo', 'Count',
-               'CollectList', 'CollectSet', 'AggregateExpression')
-@incompat
 @pytest.mark.parametrize('data_gen', _gen_data_for_collect_op, ids=idfn)
 def test_hash_groupby_collect_with_multi_distinct_fallback(data_gen):
```

Comment:
Can we change the name of the test? It is not fallback back any more.

Reply:
Fixed.